operator/Makefile

```diff
@@ -75,6 +74,3 @@ ifeq (, $(shell which controller-gen))
 	rm -rf $$CONTROLLER_GEN_TMP_DIR ;\
```

Comment:
Hmm, I guess we need all this temporary directory dance because otherwise `go` will happily modify our `go.mod`, which we definitely don't want. Bleh.